### PR TITLE
Release v2.20.0

### PR DESCRIPTION
**Pull request by zhiqiangjin**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,69 @@
...
+- ``software_version``
+- ``peering_ha_software_version``
+- ``image_version``
+- ``peering_ha_image_version``
+2. Implemented new data source **aviatrix_gateway_image**
+3. Implemented support for preallocated IP for Azure in **aviatrix_gateway** with the following attributes:
+- ``eip``
+- ``ha_eip``
+- ``azure_eip_name_resource_group``
+- ``ha_azure_eip_name_resource_group``
```
Comment (inc1t3Ful) [inline CHANGELOG.md]: ``peering_ha_azure_eip_name_resource_group``

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,69 @@
...
+4. Updated the format for ``remote_vpc_name`` in **aviatrix_transit_external_device_conn** for BGP over LAN connections to "<vnet_name>:<vnet_resource_group>:<subscription_id>"
+
+#### CloudWAN
+1. Implemented support for Selective Gateway Upgrade in **aviatrix_device_registration** when used for CloudN as a Gateway with the following new attributes:
+- ``software_version``
+- ``is_caag``
+
+#### Useful Tools
+1. Implemented cross subscription support for **aviatrix_vpc** in Azure by updating the following attribute:
+- ``vpc_id``
```
Comment (inc1t3Ful) [inline CHANGELOG.md]: Should we mention that the new 3-tuple format explicitly? (and its format)

File: CHANGELOG.md (hunk 1 above)
Comment (mlin-aviatrix) [reply CHANGELOG.md]: Also `peering_ha_eip`

File: CHANGELOG.md (hunk 2 above)
Comment (inc1t3Ful) [reply CHANGELOG.md]: 1. Implemented cross-subscription support for **aviatrix_vpc** for Azure by updating ``vpc_id`` to the new following 3-tuple format: ``<vnet-name>:<resource-group-name>:<GUID>``

**Review by inc1t3Ful — APPROVED** [review]

**Pull request merged**